static/js/TopicLandingPage/TopicsLandingPage.jsx

```diff
@@ -12,8 +14,11 @@ export const TopicsLandingPage = ({openTopic}) => {
         <div className="readerNavMenu" key="0">
             <div className="content">
                 <div className="sidebarLayout">
-                    <div className="contentInner mainColumn">
+                    <div className="contentInner mainColumn topic-lading-page-content">
                         <TopicLandingSearch openTopic={openTopic} numOfTopics={5000}/>
+                        <RainbowLine rainbowClassname={"topic-landing-upper-rainbow"}/>
```

Comment:
Include RainbowLine in TopicSalad component. They are logically meant to be rendered together.